src/cls/fifo/cls_fifo.cc

```diff
@@ -16,6 +16,7 @@
 
 #include "include/buffer.h"
 #include "include/types.h"
+#include "common/Formatter.h"
```

Comment:
nit: should be sorted alphabetically (`c` precedes `i`).

Reply:
Done